frontend/src/store/useTaskStore.js

```
@@ -0,0 +1,103 @@
+
+  
+  createTask: async (taskData) => {
+    set({ isCreatingTask: true, taskError: null });
+    try {
+      const response = await axiosInstance.post("/tasks", taskData);
+      set((state) => ({
+        tasks: [...state.tasks, response.data],
+      }));
+    } catch (error) {
+      console.error("Error creating task:", error);
+      set({ taskError: "Failed to create task: " + error.message });
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)